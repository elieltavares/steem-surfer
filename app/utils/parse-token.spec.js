/* eslint-disable */

import parseToken from './parse-token';

describe('Parse token', () => {
  it('(1) should parse', () => {
    const input = '18.494 SBD';
    expect(parseToken(input)).toMatchSnapshot();
  });

  it('(2) should parse', () => {
    const input = '0.000 STEEM';
    expect(parseToken(input)).toMatchSnapshot();
  });

  it('(2) should parse number', () => {
    const input = 0;
    expect(parseToken(input)).toMatchSnapshot();
  });
});
