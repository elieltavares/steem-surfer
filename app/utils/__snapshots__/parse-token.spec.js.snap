// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Parse token (1) should parse 1`] = `18.494`;

exports[`Parse token (2) should parse 1`] = `0`;

exports[`Parse token (2) should parse number 1`] = `0`;
